src/data_structures/Translocation.cpp

```diff
@@ -236,27 +314,53 @@ bool Window::computeVariations(int chr2) {
 		vector<double> statisticsFirstWindow =computeStatisticsA(bamFileName, eventReads[chr2].front().RefID, startchrA, stopchrA, (stopchrA-startchrA), indexFile);
 		double coverageRealFirstWindow	= statisticsFirstWindow[0];
 		int linksFromWindow=int(statisticsFirstWindow[1]);
-		double coverageRealSecondWindow=computeCoverageB(chr2, startSecondWindow, stopSecondWindow, (stopSecondWindow-startSecondWindow) );
-	
-		//I need to find the window that maximise this
+		int averageReadLength=int(statisticsFirstWindow[2]);
+		double coverageRealSecondWindow=computeCoverageB(chr2, startSecondWindow, stopSecondWindow, (stopSecondWindow-startSecondWindow+1) );
+
 		if(pairsFormingLink >= minimumPairs  ) { //ration between coverage  // and coverageRealFirstWindow < 5*this->meanCoverage
-	
-	
+		//I need to find the window that maximise this
+			int secondWindowLength=(stopSecondWindow-startSecondWindow+1);
+			int firstWindowLength=stopchrA-startchrA+1;
+			float expectedLinksInWindow = ExpectedLinks(firstWindowLength, secondWindowLength, estimatedDistance, mean_insert, std_insert, coverageRealFirstWindow, averageReadLength);
 			if(this->chr == chr2) {
```

Comment:
@J35P312 can you remove this if here, is something I likely forgot some time ago. It is anough for you to make the changes, commit and push them and also this pull request will be updated